app/Http/Requests/QuoteProposal/CreateQuoteProposalRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Requests\QuoteProposal;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateQuoteProposalRequest extends FormRequest
```

Comment:
```suggestion
class ReviewQuoteProposal extends FormRequest
```

That should be the name isn't? 🤔